### DISCUSSION TITLE
遭遇了糟糕的bug，请救救我！

### DISCUSSION THREAD

**Discussion by VansamaZWT** [liusy58/NJU_AdvancedProgramming_SP22#193] [Q&A]
使用QT进行编译的时候，遭遇了如下的bug
![image](https://user-images.githubusercontent.com/100075314/167530596-240b0d55-5077-455f-b983-d6c4c8406293.png)
这个bug在编译阶段发生，程序尚未开始运行，对于现阶段的我而言，这种情况下的bug我只能用肉眼观察，而事实上，我的肉眼观察也没能发觉问题所在。
经互联网查询，这一报错有可能是存在没有实现的声明，但我找遍，没有发现。
希望有知有德的大家能给予一些指点。

**Comment by liusy58** [comment]
思路：二分debug，一点点缩小范围

**Reply by VansamaZWT** [reply]
问题在于我的程序直接无法运行，我无法缩小范围

**Reply by liusy58** [reply]
I mean 你每次注释一部分代码，看看那些被注释掉了之后就可以正常编译

**Reply by VansamaZWT** [reply]
谢谢您的忠告，我已经以此注释过主函数中每一行代码，结果没有发生改变，我想再找不到bug我或许要放弃了:(

**Reply by liusy58** [reply]
可以先试一下官网给的demo能不能跑起来，说不定是库什么的没装好

**Reply by VansamaZWT** [reply]
我可以在QQ上私聊您详细描述一下我的病症吗
